app/package.json

```diff
@@ -26,7 +27,10 @@
     "react": "17.0.2",
     "react-i18next": "^11.12.0",
     "react-native": "0.65.1",
+    "react-native-calendar-picker": "^7.1.2",
+    "react-native-calendars": "^1.1284.0",
     "react-native-date-picker": "^4.2.1",
+    "react-native-datepicker": "^1.7.2",
```

Comment:
Is this one being used now or could it be removed?  (and the reference in `app/globals.d.ts`?)

Reply:
Removed!